pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -264,7 +265,20 @@ func (l *LibvirtConnection) GetAllDomainStats(statsTypes libvirt.DomainStatsType
 	}
 	return domStats, nil
 }
+func (l *LibvirtConnection) GetQemuVersion() (string, error) {
+	version, err := l.Connect.GetVersion()
+	major := version / 1000000
+	version = version - (major * 1000000)
```

Comment:
Please add a comment why this works. (int != float)

Reply:
Done!